### PR TITLE
convert to origami 2.0 component

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,7 +190,7 @@ There are several options used to change the appearance or behaviour of o-banner
 
 ## Sass
 
-As with all Origami components, o-banner has a [silent mode](http://origami.ft.com/docs/syntax/scss/#silent-styles). To use its compiled CSS (rather than using its mixins with your own Sass) set `$o-banner-is-silent: false;` in your Sass before you've imported the o-banner Sass.
+As with all Origami components, o-banner has a [silent mode](http://origami.ft.com/docs/syntax/scss/#silent-styles). To use its compiled CSS (rather than using its mixins with your own Sass) call `@include oBanner();` in your Sass after you've imported the o-banner Sass.
```
Comment (notlee) [inline README.md:193]:
```suggestion
Include the "primary mixin"`oBanner` to output all `o-banner` styles for all banner variants.

```scss
@import 'o-banner/main';

@include oBanner();```
```

File: README.md [hunk 2]
```diff
@@ -190,7 +190,7 @@ There are several options used to change the appearance or behaviour of o-banner
...
 
 o-banner includes a primary mixin that takes an `$opts` parameter. The `themes` key of `$opts` includes styles for [themes](#themes). The `layouts` key includes styles for [layouts](#layouts):
```
Comment (notlee) [inline README.md:195]:
```suggestion
To output styles only for the variants of `o-banner` your project uses, the "primary mixin"  `oBanner` accepts an `$opts` argument. The `themes` key of `$opts` includes styles for [themes](#themes). The `layouts` key includes styles for [layouts](#layouts):
```

File: README.md [hunk 3]
```diff
@@ -312,15 +317,16 @@ You can change a lot of aspects of a banner's visual appearance. When using the
...
+⚠ maintained | 3 | 3.4 | [migrate to v3](MIGRATION.md#migrating-from-v2-to-v3) |
+╳ deprecated | 2 | 2.3 | [migrate to v2](MIGRATION.md#migrating-from-v1-to-v2) |
 ╳ deprecated | 1 | 1.7 | N/A |
 
 ## Contact
 
 If you have any questions or comments about this component, or need help using it, please either [raise an issue](https://github.com/Financial-Times/o-banner/issues), visit [#origami-support](https://financialtimes.slack.com/messages/origami-support/) or email [Origami Support](mailto:origami-support@ft.com).
 
----
+***
```
Comment (chee) [inline README.md:329]: dinkus

**Review by chee — APPROVED** [review]

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v4.0.0 :tada:

**Pull request merged**